index.html

```diff
@@ -4,81 +4,132 @@
-              ><img src="images/social_pic/github.png" alt="github link"
-            /></a>
-          </li>
-          <li class="icon">
-            <a href="#"
-              ><img src="images/social_pic/angellist.png" alt="Angel list link"
-            /></a>
-          </li>
-        </ul>
-      </div>
+    <!--main-->
+    <main id="main">
+      <!--home page-->
+      <section class="home-page">
+        <h1 class="header">Hey There.<br />I'm Melaku</h1>
+
+        <div class="main-description">
+          <h2 class="description-heading">I'm a Software Developer</h2>
+
+          <p class="description-para">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+      </section>
+
+      <!--icons-->
+      <ul class="icons-container bounce">
+        <li>
+          <a href="https://github.com/Melaku05"
+            ><img
+              src="images/social_pic/github.png"
+              alt="github"
+              class="main-icon bounce-icon"
+          /></a>
+        </li>
+        <li>
+          <a href="https://www.linkedin.com/in/melaku-eshetu-b34b36223/"
+            ><img
+              src="images/social_pic/linkedin.png"
+              alt="linkedin"
+              class="main-icon bounce-icon"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              src="images/social_pic/angellist.png"
+              alt="angellist"
+              class="main-icon bounce-icon"
+          /></a>
+        </li>
+        <li>
+          <a href="https://twitter.com/home"
+            ><img
+              src="images/social_pic/twitter.png"
+              alt="twitter"
+              class="main-icon bounce-icon"
+          /></a>
+        </li>
+        <li>
+          <a href="#"
+            ><img
+              src="images/social_pic/medium.png"
+              alt="medium"
+              class="main-icon bounce-icon"
+          /></a>
+        </li>
+      </ul>
+
+      <!--illustrations-->
+      <div class="illustration-container"><br /></div>
+    </main>
+
+    <section class="work" id="work">
+      <h1 class="work-header">My Recent Works</h1>
+
+      <hr />
     </section>
     <section class="container bg-color-2">
-      <div class="title">
-        <h2 class="title-text">My Recent Works</h2>
-        <ul>
-          <li><img src="images/horizontal_line.png" alt="line" /></li>
-        </ul>
-      </div>
       <div class="grid-layout">
-        <div class="grid-1">
+        <div class="grid-1 display-flex">
           <div class="img-hover">
             <ul>
               <li class="grid-img">
-                <img src="images/Imgplaceholder.png" alt="img-placeholder" />
+                <img src="assest/Img-Placeholder.png" alt="img-placeholder" />
```

Comment:
- [ ] The previous reviewer has mentioned that you should address the problems with your mobile version display. I fully understand and empathize with the position you are in as there is not much time, however for this project the main goal is that you use media queries correctly to ensure that you have a working desktop version design and that, crucially, your mobile design stays looking how it should. when you add a desktop version design it shouldn't affect the way your mobile design works.
- [ ] Your mobile design looks fine for the most part all you need to do is ensure that the cards are laid out vertically according to the Figma mobile design. this change shouldn't take too long. 👇


Your Design | Reference Design
-|-
![image](https://user-images.githubusercontent.com/18730106/155845568-d0209c07-00c4-425f-8712-ee8c3d71e26c.png)|![image](https://user-images.githubusercontent.com/18730106/155845592-adab2a64-0e08-4bcb-8959-f29c498e585f.png)